docs/source/host_challenge.md

```diff
@@ -162,6 +162,51 @@ Finally run the `./run.sh` script in the bundle. It will generate a `challenge_c
 
 **Congratulations!** you have submitted your challenge configuration for review and [EvalAI team](https://eval.ai/team) has notified about this. [EvalAI team](https://eval.ai/team) will review and will approve the challenge.
 
+## Host a challenge with remote evalution
```

Comment:
First step should be 

`Step 1: Set up the challenge`

`Follow [host challenge using github section](https://evalai.readthedocs.io/en/latest/host_challenge.html#host-challenge-using-github) to set up a challenge on EvalAI`.